datadog_checks_dev/datadog_checks/dev/docker.py

```diff
@@ -41,6 +38,29 @@ def compose_file_active(compose_file):
     return False
 
 
+def run_in_container(container_name, command, check=True, interactive=False):
+    """Runs a command in the given container. This is useful for WaitFor conditions in `docker_run`
+
+    :param container_name: The name of the container
+    :param command: command line to run in the container
+    :param check: Whether or not to raise an exception on non-zero exit codes. Default: ``True``
+    :param interactive: Attach to stdin. Default: ``False``
+    """
+    client = docker_client.from_env()
+    container = client.containers.get(container_name)
+    if not container:
+        raise Exception("Could not find container {}".format(container_name))
+    try:
+        command_line = ' '.join(command) if isinstance(command, list) else command
+    except TypeError as e:
+        print(command)
+        raise e
+    result = container.exec_run(command_line, stdin=interactive)
```

Comment:
we could get split stdin and stdout by passing a flag but I don't really see an use for that